code/index.html

```diff
@@ -3,28 +3,131 @@
+      
+        <div class="textarea">
+          <label for="fname">First name</label>
+          <input id="fname" type="text" name="fname"/>
+        
+          <label for="lname">Last name</label>
+          <input id="lname" type="text" name="lname" />
+
+          <label for="email">email</label>
+          <input id="email" type="email" name="email"/>
+        
+          <label for="password">Password</label>
+          <input id="password" type="password" name="password" required/>
+        </div>
+      
+   
+        <div class="radiobutton">
+          <div>
+            <input id="prof" type="radio" name="occupation" value="prof"/>
+            <label for="prof">young and restless</label>
+          </div>
+        </div>
+        
+        <div>
+          <input id="student" type="radio" name="occupation" value="student"/>
+          <label for="student">Tired and frustrated</label>
+        </div>
+      
+        <div class="checkbox">
+          <input id="agreement" type="checkbox" name="agreement"/>
+          <label for="agreement">Agree to the terms and conditions.</label>
+          <br>
+          <input id="newsletter" type="checkbox" name="news"/>
+          <label for="newsletter">Subscribe to the newsletter.</label>
+        </div>
+      
+        <button id="submit" type="submit" name="submit">SIGNUP NOW</button>
+  
+      </form>
+    </section> 
+
 
-    <section class="accordion">
-      <!-- Accordion -->
+    <!-- Accordion -->
+    <section class="accordion-container">
+    
+      <p>So nice that you have made it here! This is an online community for people who feel they are tired of everything around them including themselves. 
+        The goal with this website is to gather people who want to make a difference for oneself and others.
+        Put simply, this is a place for frustrated people to unload their feelings. </p>
+      <br>
       <h1>FAQ</h1>
+    
+      <div class="question" id="section1">Who is this for?</div>
+      <div class="answer">
+        <p> For individuals who want to express some angry feelings. </p>
+      </div>
 
-      <div class="question" id="section1">Section 1</div>
+      <div class="question" id="section2">What can I expect by signing up?</div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p> Get interactive with other members and life coaches. Share your story and get motivational help when you need it.
+        </p>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
+      <div class="question" id="section3">Does it cost to be a member?</div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p> The membership is free but we do take a small charge for private sessions.  
+        </p>
       </div>
+  
     </section>
-    <!-- dont forget to link the javascript file here! -->
+  </main>
+
+    
+  <!-- dont forget to link the javascript file here! -->
```

Comment:
This comment could be deleted or changed to `<!-- Link to JavaScript file here -->`